lib/chef/provider/package/rubygems.rb

```diff
@@ -401,7 +401,7 @@ def initialize(new_resource, run_context=nil)
         end
 
         def is_omnibus?
-          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef)/embedded/bin!
+          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef|chefdk)/embedded/bin!
```

Comment:
@danielsdeleo I have no idea.  But I say we update the regex to `%r!\/(opscode|chef|chefdk)\/embedded\/bin!` to handle all cases